server/scripts/2021TaxSeasonDataScript.php

```diff
@@ -0,0 +1,420 @@
+			$sixthAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 7.5, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Nebraska Union data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertAsianCommunityAndCulturalCenterData() {
+	GLOBAL $DB_CONN;
+	
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Asian Community and Cultural Center data has already been inserted');
+	}
+
+	$siteId = insertSite("Asian Community And Cultural Center", "", "");
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Wednesdays 11-2
+		$wednesdayDates = getWeeklyDatesFromRange('2021-02-03', '2021-04-07');
+		foreach ($wednesdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 11:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
+		}
+		
+		// Thursdays 11-2
+		$thursdayDates = getWeeklyDatesFromRange('2021-02-04', '2021-04-08');
+		foreach ($thursdayDates as $date) {
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 11:00:00", 6, 6, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 6, 6, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 6, 6, $siteId);
+		}
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Asian Community and Cultural Center data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertInternationalStudentScholarSiteData() {
+	GLOBAL $DB_CONN;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The International Student Scholar data has already been inserted');
+	}
+
+	$siteId = 4; // Manually obtained from PROD DB
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		// Tuesdays 12-5
+		$tuesdayDates = getWeeklyDatesFromRange('2021-03-02', '2021-04-06');
+		//TODO make sure these are 12-4
+		foreach ($tuesdayDates as $date) {
+
+			// Virtual China
+			$firstAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 10, 7, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 10, 7, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 10, 7, $siteId);
+			$fourthAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 10, 7, $siteId);
+
+			// Virtual India
+			$firstAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 5, 8, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 5, 8, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 5, 8, $siteId);
+			$fourthAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 5, 8, $siteId);			
+		
+			// Virtual Treaty
+			$firstAppointmentTimeId = insertAppointmentTime("$date 12:00:00", 5, 9, $siteId);
```

Comment:
As an aside, I think I'd probably make variables for these appointmentTypeIds. Might make it look a little better than just seeing the numerals `8`, `9`, etc. Lot easier to read `$VIRTUAL_INDIA`, etc. Not worth a change, just a thought